Resources/Computing/PoolComputingElement.py

```diff
@@ -87,12 +107,22 @@ def submitJob( self, executableFile, proxy, **kwargs ):
       pilotProxy = ret['Value']['path']
     self.log.notice( 'Pilot Proxy:', pilotProxy )
 
+    kwargs = { 'UseSudo': False }
+    if self.useSudo:
+      for nUser in range( 16 ):
```

Comment:
Then maybe we could use a constant with explicit name or something like that, or at least a comment ?